### PR TITLE
add feat to get a gworkspace authorization link

### PR DESCRIPTION
**Pull request by Isafagioli-Tenchi**

### CODE REVIEW

File: zanshinsdk/client.py [hunk 1]
```diff
@@ -921,6 +921,14 @@ def check_organization_scan_target(self, organization_id: Union[UUID, str],
                              f"/organizations/{validate_uuid(organization_id)}/scantargets/"
                              f"{validate_uuid(scan_target_id)}/check").json()
 
+    def get_gworkspace_oauth_link(self) -> Dict:
+        """
+        Retrive a link to allow the user to authorize zanshin to read info from their gworkspace enviroment.
```
Comment (edpin) [inline zanshinsdk/client.py:926]: Retrieve
environment

**Review by edpin — CHANGES_REQUESTED** [review]
Also need to add a bit more info to the PR description.

File: zanshinsdk/client.py (hunk 1 above)
Comment (Isafagioli-Tenchi) [reply zanshinsdk/client.py:926]: you mean in the docstring?!

Comment (edpin) [reply zanshinsdk/client.py:926]: Yeah. The spelling is wrong, that's all.

Comment (Isafagioli-Tenchi) [reply zanshinsdk/client.py:926]: oh ok, I'll correct that but just so you know I copied from the api (exactly the same) so it should be corrected there as well!

Comment (edpin) [reply zanshinsdk/client.py]: Yikes. Do you know where it lives? Maybe you can correct that there too. Thanks.

**Review by edpin — APPROVED** [review]
Need a PR description too.

**Pull request merged**